django/template/base.py

```diff
@@ -722,7 +723,11 @@ def resolve(self, context, ignore_failures=False):
                         obj = string_if_invalid
         else:
             obj = self.var
+        seen_escape_filter = False
+        escape_isnt_last_filter = True
         for func, args in self.filters:
+            if seen_escape_filter:
+                escape_isnt_last_filter = False
```

Comment:
I might be missing something but I can't come up with a way `escape_isnt_last_filter is True` when reaching this branch as the only way to get in there is when `seen_escape_filter is True` which is only set in the `isinstance(obj, EscapeData)` branch below where `escape_isnt_last_filter` is immediately set to `False`.